patterns/sso-iam-identity-center/aws-auth/main.tf

```diff
@@ -49,6 +49,14 @@ module "eks" {
     }
   }
 
+  tags = local.tags
+
+}
+
+module "aws_auth" {
```

Comment:
Since cluster access entry is the long term vision and recommended approach - is there any reason why we should keep the `aws-auth` version? For example, its already planned in v21 of the EKS module to remove ALL `aws-auth` components, in v20 we're providing just enough to support the transition

Reply:
I agree with that. However there are still customers relying on the `aws-auth`, and are still in the migration phase. We can drop it after is fully obsolete, what do you think?

Reply:
> However there are still customers relying on the aws-auth, and are still in the migration phase. 

Customers are still on 1.21 and still in the migration phase, but we only show 1.29 😉